promql/parser/generated_parser.y

```diff
@@ -641,18 +666,42 @@ series_item     : BLANK
                 | series_value TIMES uint
                         {
                         $$ = []SequenceValue{}
+                        // add an additional value for time 0, which we ignore in tests
```

Comment:
Nit: Throughout the Prometheus code base, we prefer correct English syntax in comments. (I.e. start sentence with a capital letter, end with a period. Apply everywhere.)